packages/core/src/tracing/spanBuffer.js

```diff
@@ -483,3 +501,28 @@ function removeSpansIfNecessary() {
     spans = spans.slice(-maxBufferedSpans);
   }
 }
+/**
+ * @param {import('../core').InstanaBaseSpan} span
+ * @returns {import('../core').InstanaBaseSpan} span
+ */
+function processSpan(span) {
+  if (!span || typeof span.n !== 'string') {
+    return span;
+  }
+
+  // Currently, filter only if the ignoreEndpoint is configured.
+  // In the next phase, this check will be removed.
```

Comment:
qs: I do not understand this comment. Why will this check be removed? We only filter if the customer wants to filter.

Reply:
removed